lib/ex_doc/formatter/html/templates/module_template.eex

```diff
@@ -14,7 +14,7 @@
 
       <%= if deprecated = module.deprecated do %>
         <div class="deprecated">
-          This <%= module.type %> is deprecated. <%= h(deprecated) %>.
+          <%= translate(config, "This") %> <%= module.type %> <%= translate(config, "is deprecated.") %> <%= h(deprecated) %>.
```

Comment:
You need to translate the whole sentence. 'module.type' is not being translated and if module.type is "function" which is femenine "this" and "deprecated" won't match the gender.
Sorry if.this is not the case, I'm in my phone,

Reply:
I am going to change the hole sentence and make a single call to translate function instead of 2 or 3 calls.